vectory/visualization/utils.py

```diff
@@ -47,12 +47,15 @@ def pca_plus_umap_calc(embeddings):
 
 
 def calculate_points(model, embeddings, rows):
-    if model == "UMAP":
-        dim_2_points = umap_calc(embeddings)
-    elif model == "PCA":
-        dim_2_points = pca_calc(embeddings)
-    elif model == "PCA + UMAP":
-        dim_2_points = pca_plus_umap_calc(embeddings)
+    if embeddings.shape[1] == 2:
```

Comment:
I am getting this error when changing the similarity function on the demo data

`AttributeError: 'NoneType' object has no attribute 'shape'`